source/agent/video/videoTranscoder/VideoTranscoderWrapper.cpp

```diff
@@ -176,6 +176,26 @@ void VideoTranscoder::forceKeyFrame(const v8::FunctionCallbackInfo<v8::Value>& a
   me->forceKeyFrame(outStreamID);
 }
 
+void VideoTranscoder::setMaxResolution(const v8::FunctionCallbackInfo<v8::Value>& args){
```

Comment:
This function is not registered in node layer through "init" function in line 24. Who will call this function?